pyglet/image/__init__.py

```diff
@@ -376,6 +403,8 @@ def save(self, filename=None, file=None, encoder=None):
                 are tried.  If all fail, the exception from the first one
                 attempted is raised.
 
+        :rtype: None or bytes
```

Comment:
:rtype: and :return: are not needed, because they are implied by the typing definitions.